ast-visual-studio-extension/CxExtension/Commands/AddExtensionWindowCmd.cs

```diff
@@ -42,16 +44,27 @@ public static async Task InitializeAsync(AsyncPackage package)
             Instance = new CxWindowCommand(package, commandService);
         }
 
-        private void Execute(object sender, EventArgs e)
+        public void Execute(object sender, EventArgs e)
         {
             _ = this.package.JoinableTaskFactory.RunAsync(async delegate
-              {
-                  ToolWindowPane window = await this.package.ShowToolWindowAsync(typeof(CxWindow), 0, true, this.package.DisposalToken);
-                  if ((null == window) || (null == window.Frame))
-                  {
-                      throw new NotSupportedException("Cannot create tool window");
-                  }
-              });
+            {
+                try
+                {
+                    await Task.Delay(1000);
```

Comment:
why do we need this delay? better to add a comment to explain it

Reply:
done